### DISCUSSION TITLE
Move Tailwind `:hover`, `:focus-visible`, and other state selectors to the stylesheet

### DISCUSSION THREAD

**Discussion by psirenny** [resend/react-email#1338] [Ideas]
### Goals

Tailwind's state selectors like `:hover` and `:focus-visible` don't work when inlined. Instead, those Tailwind classes should be moved to the stylesheet.

### Background

I think [Maizzle](https://maizzle.com) places those selectors in a `<style />` element, but I have to verify.

### Proposal

The state selectors should move to the `<style />` element the same way that the media selectors are.

**Comment by gabrielmfern** [comment]
Hmm, yeah, I suppose this would be a nice-to-have. We didn't think of this, but the structure for the Tailwind component is certainly moving in a direction that would make this easier. 

Would love to see PRs on this if you're interested in contributing!

**Comment by Sjoertjuh** [comment]
I tried implementing this, check out the PR here: https://github.com/resend/react-email/pull/1864

**Comment by ayhanap** [comment]
GJ on the PR, I am migrating from Maizzle and this is such a needed thing to work with Tailwind. Testing with the canary 💯
